.github/workflows/build-website.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
     - name: 'Upload website build'
       if: matrix.branch != 'deploy-download-preview'
-      uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2 # v2
+      uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v2
```

Comment:
@jcfr This has resulted in an outdated comment as it did not update `# v2` to either `# v3` or `# v3.1.2` (preferred). https://github.com/actions/upload-artifact/tags

Reply:
This should be fixed in https://github.com/Slicer/slicer.org/pull/164